lib/ocpp/v201/charge_point.cpp

```diff
@@ -3588,24 +3588,52 @@ void ChargePoint::handle_get_installed_certificate_ids_req(Call<GetInstalledCert
     this->send<GetInstalledCertificateIdsResponse>(call_result);
 }
 
+bool ChargePoint::should_reject_certificate_install(InstallCertificateUseEnum cert_type) const {
```

Comment:
Nice to make a function of this to clean up the handler :+1: 

I do always have a difficult time with the double negatives especially with `value_or` in the mix. Would it be an option to change this to `should_accept_certificate_install` instead?

Reply:
done